Added comments to classes

GlebEmelianov opened the pull request.

GlebEmelianov committed:
Added comments to classes

Fulcrum81 commented on src/main/java/zoo/Bird.java:
```diff
@@ -0,0 +1,16 @@
+        this.nameOfBird = nameOfBird;
+    }
+
+    public String getAction() {return action;};
```
Unnecessary ";" after a curly brace, please remove

Fulcrum81 commented on src/main/java/zoo/Bird.java:
```diff
@@ -0,0 +1,16 @@
+
+    public String getAction() {return action;};
+
+    public String getNameOfFish() {return nameOfBird;};
```
Unnecessary ";" after a curly brace, please remove

Fulcrum81 commented on src/main/java/zoo/Fish.java:
```diff
@@ -0,0 +1,16 @@
+        this.nameOfFish = nameOfFish;
+    }
+
+    public String getAction() {return action;};
```
Unnecessary ";" after a curly brace, please remove

Fulcrum81 commented on src/main/java/zoo/Fish.java:
```diff
@@ -0,0 +1,16 @@
+
+    public String getAction() {return action;};
+
+    public String getNameOfFish() {return nameOfFish;};
```
Unnecessary ";" after a curly brace, please remove

Fulcrum81 commented on src/main/java/zoo/Main.java:
```diff
@@ -0,0 +1,61 @@
+
+        Scanner scanner = new Scanner(System.in);
+
+        Mammal Lion = new Mammal("Lion", "roars", "earth", "air");
```
Here and below: remember the naming conventions, this is just a variable, the name should be in camelCase

Fulcrum81 reviewed (changes requested):
Conceptually: Can I create a Lion in this program that meows, breathes water and flies? I think I can, but I shouldn't be able to do that.
The certain animals should also be implemented as classes so that their actions are predefined and cannot be changed.
If you have need any additional clarification please feel free to ask

GlebEmelianov committed:
Added classes for each animal with their unique sound
Removed unnecessary ";" after a curly brace in classes Bird, Fish, Mammal
Renamed variables in Main class

GlebEmelianov commented:
> Conceptually: Can I create a Lion in this program that meows, breathes water and flies? I think I can, but I shouldn't be able to do that. The certain animals should also be implemented as classes so that their actions are predefined and cannot be changed. If you have need any additional clarification please feel free to ask

Added classes for each animal with their unique sound

GlebEmelianov committed:
Made changes in Zoo code. Added test frameworks and XML files for TestNG

GlebEmelianov commented:
Made changes in Zoo code. Added test frameworks and XML files for TestNG
